### PR TITLE
[CDAP-12475] Reskins Dataprep/Rules Engine welcome screen

### PR DESCRIPTION
**Pull request by ajainarayanan**
JIRA: https://issues.cask.co/browse/CDAP-12475

### CODE REVIEW

File: cdap-ui/app/cdap/components/RulesEngineHome/RulesEngineServiceControl/index.js [hunk 1]
```diff
@@ -89,24 +150,31 @@ export default class RulesEngineServiceControl extends Component {
             {T.translate(`${PREFIX}.benefits.title`)}
 
             <ul>
-              <li>{T.translate(`${PREFIX}.benefits.b1`)}</li>
-              <li>{T.translate(`${PREFIX}.benefits.b2`)}</li>
-              <li>{T.translate(`${PREFIX}.benefits.b3`)}</li>
-              <li>{T.translate(`${PREFIX}.benefits.b4`)}</li>
+              <li>
+                <span className="fa fa-laptop" />
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/RulesEngineHome/RulesEngineServiceControl/index.js:154]: We have `icon-laptop` in the `symbol-defs.svg`, so you can do `IconSVG` here too

**Review by tonybach — COMMENTED** [review]
One minor comment, otherwise code LGTM 👍

File: cdap-ui/app/cdap/components/RulesEngineHome/RulesEngineServiceControl/index.js (hunk 1 above)
Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/RulesEngineHome/RulesEngineServiceControl/index.js:154]: Talked @tonybach the shaded pixel ratio for this icon seemed off when using `IconSVG` so defaulted to font awesome. Need to investigate why this happens.

**Comment by ajainarayanan** [comment]
@elfenheart PR updated with padding right to text container.

**Review by elfenheart — APPROVED** [review]
👍  LGTM

**Pull request merged**